src/main/kotlin/calculator/domain/Delimiter.kt

```diff
@@ -0,0 +1,16 @@
+package calculator.domain
+
+@JvmInline
+value class Delimiter(val symbol: String) {
+
+    companion object {
+        private const val DEFAULT_SYMBOL = ",|:"
+
+        val DEFAULT_DELIMITER = Delimiter(DEFAULT_SYMBOL)
+
+        fun of(symbol: String?): Delimiter {
+            if (symbol.isNullOrEmpty()) return DEFAULT_DELIMITER
```

Comment:
if문 안의 line이 한줄이더라도, 중괄호를 써주게 되면 가독성이 더 좋아지는 것 같더라구요!

```suggestion
            if (symbol.isNullOrEmpty()) {
                return DEFAULT_DELIMITER
            }
```